doc/source/cluster/kubernetes/examples/mobilenet-rayservice.md

```diff
@@ -0,0 +1,47 @@
+(kuberay-mobilenet-rayservice-example)=
+
+# Serve a MobileNet image classifier on Kubernetes
+
+> **Note:** The Python files for the Ray Serve application and its client are in the repository [ray-project/serve_config_examples](https://github.com/ray-project/serve_config_examples).
+
+## Step 1: Create a Kubernetes cluster with Kind.
+
+```sh
+kind create cluster --image=kindest/node:v1.23.0
+```
+
+## Step 2: Install KubeRay operator
+
+Follow [this document](kuberay-operator-deploy) to install the latest stable KubeRay operator via Helm repository.
+Please note that the YAML file in this example uses `serveConfigV2`, which is supported starting from KubeRay v0.6.0.
+
+## Step 3: Install a RayService
+
```

Comment:
```suggestion

* Step 3.1: Download `ray-service.mobilenet.yaml`
```

Reply:
Unless you feel strongly about dividing it into two sections, I'd rather keep it as it is.